mantidproject/mantid
# Tie/fix/constrain in multi fitting

Corrected update of the gui for local parameters. Added constraints to the local parameters.

**To test:**

* Start the multi fit interface. Add some data, function.
* Set ties, fixes, constraints to parameters of functions of different data sets.
* Switch between data sets. Check that the gui updates correctly.
* Run a few fits selecting different current data set each time. Results must be the same regardless of the selected data set.

Fixes #21829

**Release Notes** 

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
